core/src/tools/CMakeLists.txt

```diff
@@ -75,6 +75,12 @@ if(NOT client-only)
     testfind dird_objects fd_objects bareosfind CLI11::CLI11
   )
   list(APPEND TOOLS_SBIN testfind)
+
+  if(NOT HAVE_WIN32)
```

Comment:
Is there a reason why this is not compiled on WIN32?

Reply:
There were some troubles getting it to compile (the good old `struct stat` in `compat.h`) and since we wont offer the dedup backend on windows for now, I decided to not bother to fix this.